docs/src/main/paradox/docs/getting-started/understanding-knowledge-graphs.md

```diff
@@ -1 +1,210 @@
+
+The relation describes how entities are connected. For example, being member of, visiting a place, belonging to a country...
+![Graph Structure](./assets/kg-entity_relation.png)
+
+It's worth noting that having this graph data structure makes it easy to merge new data into an existing graph by connecting new entities and new relations.
+
+### Describing knowledge in a graph
+
+After deciding what concept or idea that we wish to model in a knowledge graph, we need to collect the metadata that describes our entities.
+In order to make our data interoperable with other data and to allow us to import data from existing sources we should use a shared language when describing entities and relations.
+This is achieved using an [ontology](https://en.wikipedia.org/wiki/Ontology_%28information_science%29).
+
+The ontology is a kind of dictionary containing types that can be used to describe entities. The metadata is created by picking a type from the ontology
+that matches what you wish to describe and defining the properties of the chosen type. E.g. a person can be described by a [Person](https://schema.org/Person) type.
+The name of the person can be encoded in a [givenName](https://schema.org/givenName) property. This results in a set of data that can be processed both by humans and computers (as it is structured).
+
+For example, in the image below, you can see that the entity **Toy Story** with a property **Release date** that has the value **March 29th, 1995** and the **Director** property with the value John Lasseter.
+
+ ![Graph Structure](./assets/kg-structure0.png)
+
+### Inferring new data from existing relationships
+
+Since the entities are encoded using well-defined ontologies it allows us to draw conclusions based on the data we have and create new inferred data.
+This is done by giving the knowledge graph to an [inference engine](https://en.wikipedia.org/wiki/Inference_engine) that can apply a
+set of logical rules to the data to derive new information. For example, if we have a knowledge graph containing persons and their
+birth year we could create an inference engine with rules that determine whether a person is a child or a retired person.
+ 
+In the second example below, we have **Toy Story**, it has characters (using the relation called **Character**) and we have asserted the cast of characters.
+Having defined this, we can now define a rule that instructs the inference engine to create
+new relations **Voice Cast** whenever we find the **Cast** of a **Character**. This enables us to query the graph
+and directly get **Toy Story**'s voices cast without having to navigate through its **Cast** first.
+
+ ![Graph Structure](./assets/kg_inferred-relation.png)
+
+In reality inferring knowledge is a complex topic. [This video lecture](https://www.youtube.com/watch?v=AUtk6NBqEFY) shows some
+real-world examples of algorithms and rules for inference engines. E.g. [reasoning about a persons creditworthiness based on what
```

Comment:
probably those 2 links description should go in brackets: 
- ["reasoning about a persons creditworthiness based on what their home address is"](...)
- ["trying to spot fake papers that have been submitted to a journal"](...)

Reply:
You mean quotes? I would leave them without quotes, i.e. as they are, since I'm only describing some use cases in the video lecture. If it would be in a quote it may look like a title of a talk which is not correct 🤔 Let me know if you have any more thoughts on this.